PR #21 Reformatted (#128)
mongodb/docs-meta: source/style-guide/markup/directives/code-blocks.txt

=============
Code Examples
=============

> As needed, use the appropriate language option for code blocks directives

Reviewer: ```suggestion
As needed, use the appropriate language option for code block directives
```

> including the following:

Reviewer: ```suggestion
including the following directives:
```

- ``code-block``
- ``io-code-block``
- ``literalinclude``

> This enables syntax highlighting which makes the source code easier to

Reviewer: Define "this".

```suggestion
These directives can enable syntax highlighting, which makes the source code easier to
```
